src/CLICommand/cli_command.test.ts

```diff
@@ -0,0 +1,81 @@
+import { expect } from 'chai';
+import { Command } from 'commander';
+import { SinonStubbedInstance, stub } from 'sinon';
+import { CLICommand, CommandDescriptor } from './cli_command';
+
+// declare all parameters
+import '../parameter_declarations';
+import { DriveNameParameter } from '../parameter_declarations';
+import { CliApiObject } from './cli';
+import { baseArgv } from './test_constants';
+
+const MY_DRIVE_NAME = 'My awesome drive!';
+const driveNameCommandName = 'drive-name-test';
+const driveNameCommandDescription: CommandDescriptor = {
+	name: driveNameCommandName,
+	parameters: [DriveNameParameter],
+	action(option) {
+		/** This code here will run after argv is parsed */
+		expect(option.driveNameTest).to.equal(MY_DRIVE_NAME);
+	}
+};
+const driveNameArgv: string[] = [...baseArgv, driveNameCommandName, '--drive-name', MY_DRIVE_NAME];
+
+// const MY_DRIVE_ADDRESS = '00000000000000000000000000000000';
```

Comment:
Lets drop the commented code here, unless you're planning on extending the test with this.

Reply:
Removed unused comment 906681c